sdl/v1.go

```diff
@@ -1,14 +1,14 @@
 package sdl
 
 import (
-	"fmt"
 	"sort"
 
 	"github.com/blang/semver"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/ovrclk/akash/manifest"
 	"github.com/ovrclk/akash/types"
 	dtypes "github.com/ovrclk/akash/x/deployment/types"
+	"github.com/pkg/errors"
```

Comment:
nitpick(not necessary to fix): raise import line to be just below the standard library packages.